test/nx-zlib.conf

```diff
@@ -48,3 +48,7 @@ logfile = ./nx.log
 # range is from 0 to 100 (0% to 100% use of nx).
 # default: 100
 #nx_ratio_s = 100
+
+# Maximum number of times deflateInit/inflateInit can reuse an already-open VAS
+# window
+# max_vas_reuse_count = 100
```

Comment:
From an API point of view, it seems unnecessary to have both `dis_savedevp` and `max_vas_reuse_count`, e.g. `max_vas_reuse_count = 0` behaves as `dis_savedevp = 1`.

Can we remove `dis_savedevp`?

Reply:
@tuliom I'd be OK with the removal. My only concern was if there were any users already relying on this config.

Reply:
I don't know, but I think we can always revert it later, if necessary.

Reply:
Ok, I `dis_savedevp` has been removed.